frontend/src/components/BottomBar.js

```diff
@@ -512,29 +496,13 @@ class BottomBar extends Component {
                   onClick={this.handleClickOpenProfile}
                   sx={{ margin: 0, bottom: 17, right: 8 }}
                 >
-                  <Badge
-                    badgeContent={
-                      (this.state.active_order_id > 0) & !this.state.profileShown ? '' : null
-                    }
-                    color='primary'
-                  >
-                    <Avatar
-                      className='phoneFlippedSmallAvatar'
-                      sx={{ width: 55, height: 55 }}
-                      alt={this.props.nickname}
-                      imgProps={{
-                        onLoad: () => this.props.setAppState({ avatarLoaded: true }),
-                      }}
-                      src={
-                        this.props.nickname
-                          ? window.location.origin +
-                            '/static/assets/avatars/' +
-                            this.props.nickname +
-                            '.png'
-                          : null
-                      }
-                    />
-                  </Badge>
+                  <RobotAvatar
+                    style={{ width: 55, height: 55 }}
```

Comment:
Once again here the image size seems to not be taken. The profile button size on mobile is now the size of all other avatars:
![bottomavatar2](https://user-images.githubusercontent.com/90936742/191738685-6a921d2a-6ae0-4d51-8727-692724eae1b7.png)

Used to look like this (55 x 55):
![bottomavatar1](https://user-images.githubusercontent.com/90936742/191738609-472d4fe7-7675-423f-8c52-032e6cb67f88.png)

Reply:
That's weird for me it doesn't look like that, what's your browser??

Reply:
Anyways I think it's about the same topic, my intention is to normalize everything not only form the tech side but also UX